prime-router/src/main/kotlin/TransportType.kt

```diff
@@ -60,6 +61,33 @@ data class AS2TransportType
 ) :
     TransportType("AS2")
 
+/**
+ * FTPSTransportType
+ */
+data class FTPSTransportType
+@JsonCreator constructor(
+    val host: String,
+    val port: Int,
+    val username: String,
+    val password: String,
+    val protocol: String = "SSL", // TODO: make enum SSL/TLS
```

Comment:
Consider making this an enum

Reply:
Will do. Where in the code do enums like this get stored? Can I put it in this file?

Reply:
I think that's fine. I'd call the enum `FtpsProtocol` as opposed to plain `Protocol` to make clear that this is for FTPS only.